tests/acceptance/test_organization_document_integration_detailed_view.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import
+
+from sentry.testutils import AcceptanceTestCase
+
+
+class OrganizationDocumentIntegrationDetailView(AcceptanceTestCase):
+    """
+    As a developer, I can view an document-based integration, and learn more about it with the linked resources.
+    """
+
+    def setUp(self):
+        super(OrganizationDocumentIntegrationDetailView, self).setUp()
+        self.login_as(self.user)
+
+    def load_page(self, slug, configuration_tab=False):
```

Comment:
@NisanthanNanthakumar You don't need the logic for `configuration_tab` here

Reply:
@NisanthanNanthakumar You still have the parameter `configuration_tab`

Reply:
@scefali missed that! updated.